crates/turbo-tasks-testing/src/lib.rs

```diff
@@ -48,7 +48,7 @@ impl TurboTasksCallApi for VcStorage {
         let future = func();
         let i = {
             let mut tasks = self.tasks.lock().unwrap();
-            let i = tasks.len();
+            let i = tasks.len() + 1;
```

Comment:
Nit: Track `id` and `i` separately. We only need a `TaskId::from(i + 1)` once, instead of a `+ 1` and several `- 1`s